app/models/activity.rb

```diff
@@ -13,7 +13,7 @@ class Activity < ApplicationRecord
     :purpose_step,
     :sector_category_step,
     :sector_step,
-    :status_step,
+    :programme_status_step,
```

Comment:
Thanks for pointing this out leeky! I had no idea about the `form_state`. I think I'll need a hand with this issue due to lack of familiarity

Reply:
I also didn't realise the form state was persisted. Would it make sense to keep the existing name as `status_step` and thus not break any existing records, or would this create more confusion over terminology?

Reply:
I vote for keeping the original `status_step` name here. It's not really mentioned outside of the form handling, unlike say submissions where the rename was more necessary.

@jcoglan The form state is persisted because we have *a lot* of conditional validations which rely on it.

Reply:
I'll keep the original step name then. Thanks both!